inc/ThirdParty/Themes/ThirdpartyTheme.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Themes;
+
+use WP_Theme;
+
+abstract class ThirdpartyTheme implements \WP_Rocket\Event_Management\Subscriber_Interface {
+	/**
+	 * Name from the theme.
+	 *
+	 * @var string
+	 */
+	protected static $theme_name = '';
+	/**
+	 * Check if the theme or one of its child theme is activated.
+	 *
+	 * @param WP_Theme $theme current theme.
+	 * @return bool
+	 */
+	protected static function is_theme( $theme = null ) {
+		$theme = $theme instanceof WP_Theme ? $theme : wp_get_theme();
+
+		return ( str_contains( static::$theme_name, $theme->get_template() ) );
```

Comment:
```suggestion
		return ( str_contains( static::$theme_name, strtolower( $theme->get_template() ) ) );
```
the template can have uppercase characters, so we need to make sure it's all lower case before using `str_contains()` as the function is case sensitive